src/stores/rootStore.ts

```diff
@@ -225,40 +225,74 @@ export const useRootStore = defineStore('root', {
       return state.stateFiling
     },
 
-    /**
-     * A formatted concatenation of the name and the effective date of the filing.
-     * Only used when entity is historical.
-     */
+    /** The historical reason text to display in the info header. */
     getReasonText (): string {
+      const enDash = '–' // ALT + 0150
```

Comment:
Why are you doing this Sev? Wouldn't it be easier just to use the character in the return statement below?

Reply:
This is not a dash. It's a special [HTML entity](https://www.w3.org/wiki/Common_HTML_entities_used_for_typography) - kind of a long dash.

First, I capture it in a variable since you need to use a special keyboard command to enter it, or cut-and-paste it from somewhere.

Second, I call it out as its own variable since it's not a simple dash. And also for folks to ask me about it 😆

Reply:
Ahhh, I see. OK, sweet!

Reply:
FYI, I know the uxpin shows it as a regular dash but I think it's wrong. When we first implemented a historical reason, a longer dash was desired. I assume the amalgamation case is the same as the original cases.

Reply:
OK, makes sense. Deal.